modules/core-services/utils/regexp.go

```diff
@@ -15,7 +15,7 @@ package utils
 
 import "regexp"
 
-var orgReg = regexp.MustCompile(`^[0-9a-z]\d*[a-z-][a-z\d-]*[0-9a-z]$`)
+var orgReg = regexp.MustCompile(`^(?:[a-z]+|[0-9]+[a-z]+|[0-9]+[-]+[a-z0-9])+(?:(?:(?:[-]*)[a-z0-9]+)+)?$`)
```

Comment:
make a comment describe what this regexp means, it's too complex

Reply:
org regex modify from 
![image](https://user-images.githubusercontent.com/30427474/125055130-7b887e80-e0d9-11eb-84da-0083b3f564e8.png)
,but org name can not be pure digital

Reply:
add into code, and use english